Gemfile

```diff
@@ -1,4 +1,4 @@
 source 'https://rubygems.org'
 
 gem 'cocoapods', '1.8.4'
-gem 'cocoapods-generate', '~> 1.0'
+gem 'cocoapods-generate', git: 'https://github.com/square/cocoapods-generate', ref: '2dcee88'
```

Comment:
Can we add a comment as to why we need to use this ref and when we can go back to a normal released version?